include/seqan/system/system_event_win.h

```diff
@@ -348,6 +230,7 @@ inline bool signal(Event & e)
     return e.signal();
 }
 
+#endif  // #ifdef STDLIB_VS
 /*
```

Comment:
can you remove the commented section below this since you are cleaning up already?